### PR TITLE
Ported CC 071 Minesweeper

### PR DESCRIPTION
**Pull request by DenisovichDev**
Ported Coding Challenge 071: Minesweeper.

- Added timestamps (wasn't in the YouTube video)
- Fixed as many links as possible
- Removed the inactive projects

**Things to consider**
- There should probably be a way to open the source codes of the contributions from the website.
- Clicking on the contribution card always opens the links from the "url" property. In case the "url" property is missing, it doesn't open the YouTube video from the "videoId" property. 

I see that these things were discussed in #244

### CODE REVIEW

**Comment by shiffman** [comment]
Thank you for contributing @DenisovichDev! Showcase projects that no longer are active don't need to be included. They can always be resubmitted and they are archived on the old website (https://github.com/CodingTrain/website-archive).

**Comment by DenisovichDev** [comment]
Okay, I'll keep that in mind!

File: content/videos/challenges/71-minesweeper/index.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  ],
+  "codeExamples": [
+    {
+      "title": "Minesweeper",
+      "description": "A simple Minesweeper game",
+      "image": "minesweeper.png",
+      "urls": {
+        "p5": "https://editor.p5js.org/codingtrain/sketches/-mz-ePoqd",
+        "processing": "https://github.com/CodingTrain/Coding-Challenges/tree/main/071_minesweeper/Processing",
+        "other": "https://github.com/CodingTrain/Coding-Challenges/tree/main/071_minesweeper"
```
Comment (shiffman) [inline content/videos/challenges/71-minesweeper/index.json:39]: Noting we can skip the "other" property if it's just the same as the Processing code. Other is really just for if it's not p5 or Processing.

File: content/videos/challenges/71-minesweeper/index.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          "icon": "🎥",
+          "title": "Prototypes in JavaScript",
+          "url": "https://www.youtube.com/watch?v=hS_WqkyUah8",
+          "description": "In this video, I examine the concept of \"Prototype\" in the JavaScript programming language."
+        },
+        {
+          "icon": "🎥",
+          "title": "Inheritance with Prototype in JS",
+          "url": "https://www.youtube.com/watch?v=CpmE5twq1h0",
+          "description": "You should go and watch my videos on ES6 classes instead, but if you choose to be here, I will discuss how inheritance works with prototypes in JS."
```
Comment (shiffman) [inline content/videos/challenges/71-minesweeper/index.json:80]:
```suggestion
          "description": "You should probably watch my videos on ES6 classes instead, but if you insist on being here, I will discuss how inheritance works with prototypes in JS."
```

**Review by shiffman — COMMENTED** [review]
Fantastic, wow, thank you @DenisovichDev! I've made a few small comments. Here are some other notes.

1. I think I am going to try to get away from having redundant text in the preview image for the video (this is the file `index.jpg`. I might actually suggest just replacing it with the `minesweeper.png` image that is for the code example.

2. We've decided that the `source` property in the contributions is no longer applicable. See #244. So those can be removed. It's up to the showcase submission to include a link to the source (if they want) on the url they submit themselves. It's ok to leave them there for archival purposes, but I just want to note that they aren't seen on the site itself.

Let me know if you have any questions, thank you so much for doing this!!!

**Comment by DenisovichDev** [comment]
Thanks for all the guidance @shiffman! I have resolved all the issues and it's good to go. I'll keep the things you mentioned in mind for the next PRs.

**Comment by shiffman** [comment]
Amazing, thank you!! 🎉🎉🎉🎉

**Pull request merged**